woocommerce/woocommerce-android
# Adjust product images corners

### Description
This small PR adjusts the image corners from different screens to match the changes proposed here: pe5pgL-1us-p2#comment-1476.

### Testing instructions
Open the different screens and check that the product's image has rounded corners.

### Images/gif

| Products list  | Order detail | Refund |
| ------------- | ------------- | ------------- |
| <img width="200" src="https://user-images.githubusercontent.com/18119390/207930848-ba0d4e11-f9fa-40af-9f21-62189e5405fb.png" /> | <img width="200" src="https://user-images.githubusercontent.com/18119390/207930855-41044459-4ae7-4d31-a5cc-b2bb094467f1.png" />  | <img width="200" src="https://user-images.githubusercontent.com/18119390/207930870-014ebd4f-3b84-4020-a3b5-7d98796d1adb.png" /> |

| Top performers  | Refunded |
| ------------- | ------------- |
| <img width="300" src="https://user-images.githubusercontent.com/18119390/207930865-14e3bd55-6178-49a2-9a61-cd20e35462da.png" /> | <img width="300" src="https://user-images.githubusercontent.com/18119390/207930867-db9656b6-5963-4939-8232-14cedad52b15.png" />  |
